libc/include/llvm-libc-macros/linux/fcntl-macros.h

```diff
@@ -88,6 +88,9 @@
 // Close on succesful
 #define F_CLOEXEC 1
 
+// Close on execute for fcntl.
+#define FD_CLOEXEC 1
```

Comment:
I don't know that we should be providing this definition though. The linux kernel headers should be providing this value.